plugins/source/notion/docs/tables/notion_users_table.md

```diff
@@ -0,0 +1,17 @@
+# Table: notion_users_table
```

Comment:
These errors will resolve once the docs are moved to the Website per my comments